src/errors.ts

```diff
@@ -26,6 +26,8 @@ export const ERRORS = {
   INVALID_TYPE_ERROR: 'No such type allowed',
   DELETE_ACCOUNT_ERROR: 'Only imported accounts could be deleted',
   TOKEN_NOT_REGISTERED: 'Token not registered',
-  INVALID_KEY: 'Invalid secretKey to create identity',
+  INVALID_KEY: 'invalid-key',
```

Comment:
Its weird how you are mixing up error codes with error messages. Id separate in two constant records